source/docs/custom-manufacturing/3d-printing.rst

```diff
@@ -33,64 +33,97 @@ Disadvantages of 3D Printing
 Common Filament Types
 ---------------------
 
+For almost every part that needs to be 3D Printed for FTC, **PLA (or PLA+, Pro, etc) and PETG will meet all the needs for strength, durability, and aesthetics.** These two filament types are by far the easiest to print, and are sold by many manufacturers for reasonable prices. Most of the other filaments here offer very specific advantages (like TPU) that come at the cost of effort, time, and well, money.
```

Comment:
probably just keep it as "and money" instead of "and well, money" but this is a style choice